lib/local-links-manager/export/bad_links_url_and_status_exporter.rb

```diff
@@ -0,0 +1,27 @@
+require 'csv'
+
+module LocalLinksManager
+  module Export
+    class BadLinksUrlAndStatusExporter
+      HEADINGS = %w(url status).freeze
+
+      def self.local_authority_bad_homepage_url_and_status_csv
+        CSV.generate do |csv|
+          csv << HEADINGS
+          LocalAuthority.where.not(status: "200").distinct.pluck(:homepage_url, :status).each do |la|
+            csv << la
+          end
+        end
+      end
+
+      def self.bad_links_url_and_status_csv
+        CSV.generate do |csv|
+          csv << HEADINGS
+          Link.enabled_links.where.not(status: "200").distinct.pluck(:url, :status).each do |link|
```

Comment:
`Link.enabled_links` actually returns a different number than if we looped through each LocalAuthority and called `provided_service_links` on it. The former currently returns '40895' in Integration, the latter '39105'. I think this is due to us not taking tiers into account for `enabled_links`. I've been meaning to have a look at this but not had a chance to. We should look at this at some point in the near future. It's probably fine for this PR, as the difference isn't very large.